save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -575,7 +575,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
 
                 nav {
                     className = ClassName("nav nav-tabs")
-                    OrganizationMenuBar.entries
+                    OrganizationMenuBar.values()
```

Comment:
Because in past commits I accidentally changed `values()` ​​to `entries` based on a suggestion from IDEA, without thinking about the fact that `entries` are now immutable and in some cases the logic may be broken. Therefore, more thorough analysis is required, but for now I decided to return `values()`.